pykafka/pykafka/protocol.py

```diff
@@ -101,6 +101,11 @@ def raise_error(self, err_code, response):
                 clsname, response))
 
 
+class PartitionResponse(object):
```

Comment:
This should go away. All of the classes that inherit from it should be converted to namedtuples. Functionally, it's the same thing, and it avoids the overhead of instantiating classes.